UPGRADE-4.0.md

```diff
@@ -21,6 +21,9 @@ Please be aware the config
 ```
 sonata_block:
         context_manager: sonata.page.block.context_manager
+
+sonata_page:
```

Comment:
Imo it should have its own section inside the upgrade file, this sections talks about SonataBlockBundle

Reply:
Can you check again @jordisala1991